src/services/SupabaseDataService.ts

```diff
@@ -197,4 +198,26 @@ export class SupabaseDataService {
       .eq("id", orderId)
       .single();
   }
+
+  async upsertUsers(
+    users: DataDatabase["public"]["Tables"]["users"]["Insert"][],
+  ) {
+    return this.supabaseData
```

Comment:
We discussed to start using the kysely client as a replacement for supabase, is there a specific reason why this is implemented using supabase?

Filtering, sorting and pagination are handled in the baseResolver defined in `src/graphql/schemas/resolvers/baseTypes.ts`

Reply:
Resolved in https://github.com/hypercerts-org/hypercerts-api/pull/168/commits/bd5b11c3a23a262de2d68bd710baaeaff3f98a5f